LoopKit/DeviceManager/PumpManager.swift

```diff
@@ -163,6 +163,12 @@ public protocol PumpManager: DeviceManager {
     ///   - completion: A closure called after the command is complete
     ///   - error: An error describing why the command failed
     func resumeDelivery(completion: @escaping (_ error: Error?) -> Void)
+    
+    /// Notifies the PumpManager of a change in the user's preference for maximum basal rate.
+    ///
+    /// - Parameters:
+    ///   - rate: The maximum rate the pumpmanager should expect to receive in an enactTempBasal command.
+    func setMaximumTempBasalRate(_ rate: Double)
```

Comment:
Do you also need to provide a default implementation, so as not to break existing `PumpManager`s?

Reply:
Method dispatch for protocol extensions use static dispatch, which means that any clients with a generic PumpManager reference will call the default implementation, and not the overridden one.

Reply:
I need to understand better what the best practices are, then, for introducing breaking API changes such as this with respect to how it affects downstream dependents.  Probably a topic for a dev sync.

Reply:
Semantic versioning of LoopKit would require a major version bump on release. The versioning of LoopKit is a bit complicated by the existence of separate DIY and Tidepool forks. During its time on dev, though, we just need to coordinate these changes. This particular change only affects DashKit on the Tidepool side, and I am submitting those changes in tandem with these.